packages/riipen-ui/src/components/AsButton.jsx

```diff
@@ -0,0 +1,82 @@
+import clsx from "clsx";
+import PropTypes from "prop-types";
+import React from "react";
+import _JSXStyle from "styled-jsx/style";
+
+const AsButton = ({
```

Comment:
I wonder if we could do something like https://mui.com/api/button-unstyled/ or https://mui.com/api/button-base/ in terms of name (asButton seems too much like a HOC).
Also if we're using for TemplateCards, is a button the best choice?  Seems like a lot to put into a <button> but tbh I'm not sure if there's rules or conventions so that's just an opinion with no backing evidence :P

Reply:
I would agree with something like ButtonBase, seeing as we are really wrapping whatever we have in a button, rather than transforming the entity into a button